packages/office-ui-fabric-react/src/components/Pivot/Pivot.types.ts

```diff
@@ -95,8 +95,6 @@ export interface IPivotStyleProps {
    */
```

Comment:
Any reason not to do the `Required<Pick<....>> & Pick<...> & {}` pattern here like you are doing in TeachingBubble?

Reply:
No, same situation as yours. It was already in. 😆

Reply:
I had actually done the changes to TextField but that's in limbo for now.. I'll update and merge